public/app/plugins/panel/dashgen/loader.ts

```diff
@@ -0,0 +1,96 @@
+import { DashboardLoaderSupport } from 'app/features/dashboard/types';
+import { DashboardDTO } from 'app/types';
+import { of } from 'rxjs';
+
+export const loader: DashboardLoaderSupport = {
+  loadDashboard: (path: string) => {
+    console.log('LOAD', path);
+    return of(superSimple as DashboardDTO);
```

Comment:
@sdboyer -- this example just loads a static definition, but if you have a simple HTTP server running we can load that dynamically